STYLEGUIDE.md

```diff
@@ -0,0 +1,11 @@
+# Style Guide
+
+The SAP Cloud SDK for AI follow the style guide from the SAP Cloud SDK for JavaScript.
+
+## Code
+
+[SAP Cloud SDK style guide](https://github.com/SAP/cloud-sdk-js/blob/main/STYLEGUIDE.md)
```

Comment:
[nitpick] Just noticed that it's written differently. Since we say SAP Cloud SDK documentation style guide below, should this also be SAP Cloud SDK code style guide?